x-pack/legacy/plugins/uptime/common/runtime_types/ping/histogram.ts

```diff
@@ -28,3 +28,15 @@ export interface HistogramResult {
   histogram: HistogramDataPoint[];
   interval: string;
 }
+
+export interface HistogramQueryResult {
+  key: number;
+  key_as_string: string;
+  doc_count: number;
+  down: {
+    doc_count: number;
+  };
+  up: {
+    doc_count: number;
+  };
```

Comment:
can this be simplified? down and up can just be numbers, do we need both key and key_as_string?

Reply:
It could be simplified, but I don't think we need to do it here. This type wasn't added in this patch, it's been moved out of [a file](https://github.com/elastic/kibana/pull/59392/files#diff-28daf0275afc736c65f3e3785801be7eL62) I deleted because it's still in use.

Reply:
I created an issue to track this https://github.com/elastic/uptime/issues/168.